cumulus/primitives/parachain-inherent/src/mock.rs

```diff
@@ -61,6 +61,8 @@ pub struct MockValidationDataInherentDataProvider<R = ()> {
 	pub raw_downward_messages: Vec<Vec<u8>>,
 	// Inbound Horizontal messages sorted by channel
 	pub raw_horizontal_messages: Vec<(ParaId, Vec<u8>)>,
+	// Additional keys to be injected
```

Comment:
```suggestion
	// Additional keys value pairs that should be injected.
```

Reply:
Typo in the suggestion. Should be
```suggestion
	// Additional key-value pairs that should be injected.
```

Reply:
@girazoki please fix and then we can merge! Ty

Reply:
done!